### PR TITLE
Update files for v2.2.1 release

### PR DESCRIPTION
**Pull request by nuttyartist**
@guihkx let me know what do you think.

### CODE REVIEW

File: packaging/linux/debian/changelog.in [hunk 1]
```diff
@@ -4,6 +4,20 @@
 
  -- @APP_AUTHOR@ <@APP_AUTHOR_EMAIL@>  @DATE_NOW_UTC@
 
+@PROJECT_NAME_LOWERCASE@ (2.2.1) stable; urgency=low
+
+  * To sustain and further our commitment to exceptional app development, we have transitioned to a subscription-based pricing model for the Pro version
+  * Resolved a conflict within the Kanban interface: previously, editing a task would not conclude if a separate task was concurrently edited
+  * Addressed an issue where editing was mistakenly permitted within the editor while the Trash folder was active
+  * Fixed bug where icons didn't show up on some Linux distorubutions
```
Comment (guihkx) [inline packaging/linux/debian/changelog.in:12]:
```suggestion
  * Fixed bug where icons didn't show up on some Linux distributions
```

File: packaging/linux/common/notes.appdata.xml.in [hunk 2]
```diff
@@ -56,6 +56,22 @@
     </screenshots>
     <content_rating type="oars-1.1" />
     <releases>
+        <release date="2023-08-30" version="2.2.0">
```
Comment (guihkx) [inline packaging/linux/common/notes.appdata.xml.in:59]:
```suggestion
        <release date="2023-08-30" version="2.2.1">
```

File: packaging/linux/common/notes.appdata.xml.in [hunk 3]
```diff
@@ -56,6 +56,22 @@
...
+            <description>
+                <p>A new release is here with many bug fixes and improvements. To sustain and further our commitment to exceptional app development, we have transitioned to a subscription-based pricing model for the Pro version.</p>
+                <p>Improvements and bug fixes:</p>
+                <ul>
+                    <li>Resolved a conflict within the Kanban interface: previously, editing a task would not conclude if a separate task was concurrently edited.</li>
+                    <li>Addressed an issue where editing was mistakenly permitted within the editor while the Trash folder was active</li>
+                    <li>Fixed bug where icons didn't show up on some Linux distorubutions</li>
```
Comment (guihkx) [inline packaging/linux/common/notes.appdata.xml.in:66]:
```suggestion
                    <li>Fixed bug where icons didn't show up on some Linux distributions</li>
```

**Review by guihkx — CHANGES_REQUESTED** [review]
Looks good overall! Just a few minor issues.

I'd like to test something on the `debian/changelog.in` file, though.

Upon trying to install the .deb package on Ubuntu 20.04 using their app store, the changelog isn't displayed at all:

image


I'm thinking I can maybe get rid of the first 6 lines of that file and see if that fixes it (while simultaneously not breaking anything :crossed_fingers:).

**Comment by nuttyartist** [comment]
Alrighty, I'll wait fot your test and then I'll commit your changes, thanks for noticing!

**Comment by guihkx** [comment]
Okay, that didn't work at all so it's probably something else I'll have to investigate in another opportunity.

I'll revert my commit and you can finish the PR.

**Comment by nuttyartist** [comment]
Done. Let me know if alright. And I'll open a new issue for the .deb changelog problem.

**Comment by guihkx** [comment]
That one typo still remains in `notes.appdata.xml.in` :sweat_smile:

**Review by guihkx — APPROVED** [review]
LGTM!

**Pull request merged**